File: atividade_nicolas/alunos/nicolas_g1/Kairon/mmc.js
import prompt from 'prompt-sync'
const input = prompt({sigint:false})

function main(){
    const valores = input("Informe o valor de A e de B: ").split(" ").map(Number)
    const [A,B] = valores

    console.log(`O valor do mmc será ${calcularMMC(A,B)}`)

    function calcularMMC(A,B){
    let resto, x, y
      x = A
      y = B
      while(resto != 0){
        resto = x % y
        x = y
        y = resto
      }
      return (A * B) / x
    }
}
main()

File: atividade_nicolas/alunos/nicolas_g1/kleberson/mmc.py
a = int(input('Valor de A: '))
b = int(input('Valor de A: '))
mmc = a * b / 2
mmc_de_a_b = 0

while a>0 and b>0:
    if mmc % 2 == 0:
        mmc_de_a_b = mmc

    else:
        print(mmc_de_a_b) 
